typings.d.ts

```diff
@@ -0,0 +1,43 @@
+interface ITesters {
```

Comment:
I don't see any advantage in doing that.
In fact, that would make things more complicated.